### ISSUE TITLE
cfargument now accepts type=integer

### ISSUE THREAD

**Issue by pfreitag** [foundeo/cfdocs#1179]
As of CF2016+ apparently https://trycf.com/gist/dcf63d355df1063831bd485ed0453d8e/acf2016 

need to see what other types might be added and confirm when it was added.

**Comment by pfreitag** [comment]
FYI - a ticket has been added to lucee: https://luceeserver.atlassian.net/browse/LDEV-2471

**Comment by lunarnorth** [comment]
Two years on, this feature is still not documented. Is it considered safe and supported to use in CF applications?

Is there any move on Adobe's part to expand support for integer types beyond this limited use? For example, in typed arrays?

**Comment by KamasamaK** [comment]
I see there's a ticket for Lucee, but I would recommend opening a [ticket with Adobe](https://tracker.adobe.com) to have these documented.

As for whether they add them to typed arrays, that should be another ticket.

**Comment by pfreitag** [comment]
@KamasamaK I agree. Hey @mark-demoss can you create ticket(s) and post the link back here.

**Comment by mark-demoss** [comment]
Will do.

I also have an open support case and will let you know what I hear back.

**Comment by mark-demoss** [comment]
https://tracker.adobe.com/#/view/CF-4212587

**Comment by mark-demoss** [comment]
Adobe has updated their documentation:

https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-tags/tags-a-b/cfargument.html
